scalacheck/src/test/scala/magnolify/scalacheck/CogenDerivationSuite.scala

```diff
@@ -45,18 +46,16 @@ class CogenDerivationSuite extends MagnolifySuite {
     }
   }
 
+  import magnolify.scalacheck.TestArbitrary._
+  implicit val cogenUri: Cogen[URI] = Cogen(_.hashCode().toLong)
```

Comment:
Why you don't use newly implemented `TestCogen.scala`?

Reply:
Because it is the `CogenDerivationSuite` and I want to check the macros here